regulations/make.py

```diff
@@ -62,6 +62,12 @@ def main():
 
 languages = languageData.keys()
 
+max_lang_width = 0
+
+for lang in languages:
```

Comment:
`max([len(x) for x in languages])` was also possible, I just totally missed that.
I'll update with the `len(max(...))` solution which is cool.